meshtastic/tests/test_ble_init.py

```
@@ -0,0 +1,200 @@
+        if _module_matches_prefix(name, prefixes):
+            del sys.modules[name]
+    sys.modules.update(snapshot)
+
+
+@pytest.mark.unit
+class TestBLEPackageInit:
+    """Test BLE package initialization and import guard."""
+
+    def test_ble_init_imports_successfully(self) -> None:
+        """Test that ble package imports successfully when bleak is available."""
+        from meshtastic.interfaces import ble  # noqa: C0415
```

Ruff (0.15.7)

[warning] Invalid rule code in `# noqa`: C0415

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)